w3af/plugins/crawl/open_api.py

```diff
@@ -411,6 +413,14 @@ def get_options(self):
         o = opt_factory('discover_fuzzable_headers', self._discover_fuzzable_headers, d, BOOL, help=h)
         ol.add(o)
 
+        d = 'Automatic path parameter discovery for further testing'
+        h = ('By default, URLs discovered by this plugin allow for injection at path parameter'
```

Comment:
The description needs to be much better. It is almost impossible to understand what this parameter does without looking at the code.

Reply:
OK, I'll take stab at improving this. I largely based the description on the discover_fuzzable_headers setting, but I agree it could be more clear.